src/LinkDotNet.NCronJob/Configuration/SupportsConcurrencyAttribute.cs

```diff
@@ -17,5 +17,8 @@ public sealed class SupportsConcurrencyAttribute : Attribute
     /// </summary>
     /// <param name="maxDegreeOfParallelism">The maximum number of concurrent instances that can run.</param>
     public SupportsConcurrencyAttribute(int maxDegreeOfParallelism) => MaxDegreeOfParallelism = maxDegreeOfParallelism;
+
+    /// <inheritdoc />
+    public SupportsConcurrencyAttribute() => MaxDegreeOfParallelism = Environment.ProcessorCount;
```

Comment:
I thought I saw in an earlier revision that you set `Environment.ProcessCount * 4` (which I would go for, as the majority of jobs are I/O based and therefore jobs will idle by quite a lot).

Reply:
That's still there in the global concurrency limit. If we also add it here then one Job with max instances would saturate the limit. We could increase both though. This requires some pretty good stress/load testing to start to see I/O limits.

Reply:
Ahhh yes - I was sure to see it somewhere ;)

No that is fine. leave it as it is - that was my fault.